src/components/modus-wc-alert/modus-wc-alert.stories.ts

```diff
@@ -41,54 +52,54 @@ const Template: Story = {
   alert-description=${ifDefined(args['alert-description'])}
   alert-title=${args['alert-title']}
   custom-class=${ifDefined(args['custom-class'])}
+  dismissable=${args.dismissable}
   icon=${ifDefined(args.icon)}
+  role=${args.role}
   variant=${ifDefined(args.variant)}
 >
-  <modus-wc-button
-    aria-label="View messages"
-    color="secondary"
-    slot="button"
-  >View Messages</modus-wc-button>
 </modus-wc-alert>
     `;
   },
 };
 
 export const Default: Story = { ...Template };
 
-export const CloseIcon: Story = {
+export const CustomButton: Story = {
   render: (args) => {
     // prettier-ignore
     return html`
-<style>
-  modus-wc-button {
-    align-items: center;
-  }
-
-  [data-theme='modus-classic-dark'] modus-wc-icon {
-    color: #fff;
-  }
-</style>
 <modus-wc-alert
   alert-description=${ifDefined(args['alert-description'])}
   alert-title=${args['alert-title']}
   custom-class=${ifDefined(args['custom-class'])}
+  dismissable=${args.dismissable}
   icon=${ifDefined(args.icon)}
+  role=${args.role}
   variant=${ifDefined(args.variant)}
 >
   <modus-wc-button
-    aria-label="notification button"
+    aria-label="View messages"
     color="secondary"
-    size="sm"
     slot="button"
-    variant="borderless"
-  >
-    <modus-wc-icon
-      aria-label="notify icon"
-      decorative=""
-      name="close"
-    ></modus-wc-icon>
-  </modus-wc-button>
+  >View Messages</modus-wc-button>
+</modus-wc-alert>
+    `;
+  },
+};
+
+export const WithCustomContent: Story = {
+  render: (args) => {
+    // prettier-ignore
+    return html`
+<modus-wc-alert
+  id="alert-123"
+  custom-class=${ifDefined(args['custom-class'])}
+  dismissable=${args.dismissable}
+  icon=${ifDefined(args.icon)}
+  role=${args.role}
+  variant=${ifDefined(args.variant)}
+>
+  <div slot="content" id="alert-123">New custom message!</div>
```

Comment:
```suggestion
  <div slot="content">New custom message!</div>
```
elements with duplicate IDs are not the best idea